public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -800,24 +800,23 @@ block style-property-name-dart-diff
 :marked
   The component defines a `deleteRequest` property that returns an `EventEmitter`.
   When the user clicks *delete*, the component invokes the `delete()` method
-  which tells the `EventEmitter` to emit a `Hero` object.
+  telling the `EventEmitter` to emit a `Hero` object.
```

Comment:
I have no opinion on "which tells" vs. "telling", but either one should have a comma before it.